src/libostree/ostree-repo.c

```diff
@@ -5823,6 +5823,24 @@ ostree_repo_regenerate_summary (OstreeRepo     *self,
                                  g_variant_new_uint64 (GUINT64_TO_BE (g_get_real_time () / G_USEC_PER_SEC)));
   }
 
+  {
+    g_autofree char *remote_mode_str = NULL;
```

Comment:
We can land this hunk to simply add the data to the summary now as a prep patch.